cmd/libs/go2idl/generator/execute.go

```diff
@@ -78,6 +94,41 @@ func (f *file) assembleToFile(pathname string) error {
 	}
 }
 
+func (f *file) verifyFile(pathname string) error {
+	log.Printf("Verifying file %q", pathname)
+	friendlyName := filepath.Join(f.packageName, f.name)
+	b := &bytes.Buffer{}
+	et := NewErrorTracker(b)
+	f.assemble(et)
+	if et.Error() != nil {
+		return et.Error()
+	}
+	formatted, err := format.Source(b.Bytes())
+	if err != nil {
+		return fmt.Errorf("unable to gofmt the output for %q: %v", friendlyName, err)
+	}
+	existing, err := ioutil.ReadFile(pathname)
+	if err != nil {
+		return fmt.Errorf("unable to read file %q for comparison: %v", friendlyName, err)
+	}
+	if bytes.Compare(formatted, existing) == 0 {
+		return nil
+	}
+	// Be nice and find the first place where they differ
+	i := 0
+	for i < len(formatted) && i < len(existing) && formatted[i] == existing[i] {
+		i++
+	}
+	eDiff, fDiff := existing[i:], formatted[i:]
+	if len(eDiff) > 100 {
+		eDiff = eDiff[:100]
```

Comment:
Is it 100 lines or 100 chars? If it is the former, we should probably print less...